public_html/admin/controller/common/menu.php

```diff
@@ -34,6 +37,17 @@ public function main()
         $menu = new ADataset ('menu', 'admin');
         $this->data['menu_items'] = $menu->getRows();
 
+        $this->loadModel('user/user_group');
+        $this->groupID = (int)$this->user->getUserGroupId();
+        if ($this->groupID !== self::TOP_ADMIN_GROUP) {
+            $user_group = $this->model_user_user_group->getUserGroup($this->groupID);
+            $this->permissions = $user_group['permission'];
+        }
+
+        foreach ($menu_items as $item) {
```

Comment:
what value of $menu_items variable? Where is was set?

Reply:
It was supposed to be set in line 38: $this->data['menu_items'] = $menu->getRows();

I didn't apply your change in that line to:
$menu_items = $menu->getRows();